### ISSUE TITLE
I wanna drop the loss down under 2, but it doesn't work.

### ISSUE THREAD

**Issue by YanAwithU** [EdjeElectronics/TensorFlow-Lite-Object-Detection-on-Android-and-Raspberry-Pi#78]
Hello, I wanna train my custom model that only detect persons.
Python file(train.py) successfully works, but loss is not.
Now, it is over 140,000 steps, loss is still over 2.(Sometimes it is under 2)
I put 100 images in test folder, 1000 images in train folder.
All images are from coco dataset.
I used ssd mobilenet v2 quantized 300x300 coco as pre-trained model.
Here is my situation.
GPU : GTX 1070
CUDA : 10.0
cuDNN : 7.4
Tensorflow version : 1.15.0 gpu
